Naprawiono podgląd potrawy w modalu

QQLKAA opened the pull request:
- przeniesiono generowanie podglądu do innej funkcji
- podgląd jest teraz skalowany do rozmiaru diva #preview

QQLKAA committed:
Naprawiono podgląd potrawy w modalu

- przeniesiono generowanie podglądu do innej funkcji
- podgląd jest teraz skalowany do rozmiaru diva #preview

Nienormalny commented on main.js:
```diff
@@ -791,6 +791,40 @@ window.onload = () => {
+        img.width = width;
+        img.height = height;
+        
+        console.log(targetWidth, img.width, img.height, scale);
```
Proszę nie zapominać o usuwaniu `console.log` jeśli jest niepotrzebne :)

Nienormalny commented on main.js:
```diff
@@ -791,6 +791,40 @@ window.onload = () => {
+    const showPreview = (img) => {
+        const preview = document.getElementById('preview');
+        let targetWidth = window.getComputedStyle(preview).width;
+        // cut "px" from width string
```
Proszę napisać komentarz w języku polskim :) "Polacy nie gęsi i swój język mają"

Nienormalny reviewed (commented):
Kod wygląda dobrze i działa. Proszę jednak o dokonanie kilku zmian w kodzie.

**Można go zredukować z 34 linijek do 25 aby nie używać dodatkowych obliczeń:**
```javascript
    const showPreview = (img) => {
        const preview = document.getElementById('preview');
        let targetWidth = window.getComputedStyle(preview).width;
        // Wytnij "px" z wartosci szerokosci (String)
        targetWidth = Number(targetWidth.substr(0, targetWidth.length - 2));
        
        const scale = targetWidth / img.width;
        img.width *= scale;
        img.height *= scale;

        if (img.height > imageOptions.maxHeight) {
            const scale = imageOptions.maxHeight / img.height;
            img.width = scale;
            img.height = scale;
        }

        const canvas = document.createElement('canvas');
        canvas.width = img.width;
        canvas.height = img.height;

        const ctx = canvas.getContext('2d');
        ctx.drawImage(img, 0, 0, img.width, img.height);

        preview.appendChild(canvas);
    }
```

QQLKAA committed:
Poprawiono kod funkcji showPreview

- Usunięto niepotrzebne zmienne width i height
- Usunięto console.log
- Zmieniono język komentarza na polski
